bindings/pyroot_legacy/inc/TPyDispatcher.h

```diff
@@ -0,0 +1,165 @@
+// Author: Wim Lavrijsen   Aug 2007
```

Comment:
This file `[bindings/pyroot_legacy/inc/TPyDispatcher.h](https://github.com/root-project/root/pull/12259/files#diff-20a2b13c04fc7c986db01214d4f1fecb4834260c6ee54689067ce489f93fc6a2)` has been removed from the master and is inadvertently being re-introduced here.